### DISCUSSION TITLE
抱脸docker部署的汉化版ChatSydney点击发送后无法收到回答

### DISCUSSION THREAD

**Discussion by 2060320350** [viopsa233/AI-testers-Forum#97] [交流]
前几日看见了大佬汉化的项目，赶紧在抱脸上部署了一个，本人刚刚接触docker，不知道什么原因，由点击发送后提示{“isTtrusted”:true},并且无法收到回答

**Comment by renqabs** [comment]
public下修改index.html,有一个ws开头的链接，把ws改为wss应该就没这个问题了
websocket = new WebSocket(ws://${window.location.host}/ws/) 改为 websocket = new WebSocket(wss://${window.location.host}/ws/)

**Reply by 2060320350** [reply]
感谢回答，我去试试
